lib/api/expect/cookie.js

```diff
@@ -46,46 +46,8 @@ class ExpectCookie extends BaseExpect {
     return `Expected cookie ${cookieName} to${negate ? ' not' : ''}`;
   }
 
-  command() {
-    return this.transportActions.getCookieString();
-  }
-
-  handleCommandPromise(promise) {
```

Comment:
Why is this function removed ?

Reply:
Because we are overriding this method here. Previously, we used to get the response as an array of cookies, where we use to filter the desired cookie, but now we don't need to filter any cookie, as we will get the named cookie, or null if there is no such cookie as a response.
 
https://github.com/nightwatchjs/nightwatch/blob/15b898cbb691881355fefe817db8127fb14ae545/lib/api/expect/_baseExpect.js#L166